exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,15 @@
 let name = "Daniel";
 let danielsRole = "mentor";
 
+function roleChecker(role) {
+  if (role === "mentor") {
+    return "Hi, I'm Daniel, I'm a mentor.";
+  } else {
+    return "Hi, I'm Daniel, I'm a student.";
+  }
```

Comment:
could you change this code to use [interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)?